samples/benchmarks/coremark/README.rst

```diff
@@ -171,18 +167,33 @@ CONFIG_APP_MODE_FLASH_AND_RUN - Start CoreMark sample automatically after flashi
    Otherwise, it will wait for the button press.
 
 .. note::
-   The :kconfig:option:`CONFIG_APP_MODE_FLASH_AND_RUN` Kconfig option is always enabled for the PPR core.
-   This core does not use buttons.
+   The :kconfig:option:`CONFIG_APP_MODE_FLASH_AND_RUN` Kconfig option is always enabled for the PPR core on the ``nrf54h20dk/nrf54h20/cpuapp`` board target.
+   This core on the ``nrf54h20dk/nrf54h20/cpuapp`` board target does not use the on-board buttons and LEDs.
+
+.. _SB_CONFIG_APP_CPUFLPR_RUN:
+
+SB_CONFIG_APP_CPUFLPR_RUN - Enable the benchmark execution also for the FLPR core
```

Comment:
\`\`SB_CONFIG_APP_CPUFLPR_RUN\`\`

Reply:
This is a change of documentation style. I would like to get some feedback from @peknis, @divipillai or @annwoj

Reply:
Here we are defining the Kconfig (Sample-specific) options. Hence, the style should be kept as it. This is how it renders - https://ncsdoc.z6.web.core.windows.net/PR-20462/nrf/samples/benchmarks/coremark/README.html#configuration-options.

Reply:
Not how I expected, marking as resolved